Engine/razix_engine.lua

```diff
@@ -251,6 +260,19 @@ project "Razix"
             "vendor/glad/src/glad.c"
         }
 
+        -- Option to use Windows native input instead of GLFW
+        newoption {
+            trigger = "use-windows-input",
+            description = "Use Windows native input API instead of GLFW for input handling"
+        }
+
+        -- Conditionally include Windows input if option is specified
+        if _OPTIONS["use-windows-input"] then
+            removefiles { "src/Razix/Platform/GLFW/GLFWInput.h", "src/Razix/Platform/GLFW/GLFWInput.cpp" }
+            files { "src/Razix/Platform/Windows/WindowsInput.h", "src/Razix/Platform/Windows/WindowsInput.cpp" }
+            defines { "RAZIX_USE_WINDOWS_INPUT" }
+        end
+
```

Comment:
Perfect! bnut move this snippett of new option to premake-config.lua in scripts folder